graphql-jpa-query-schema/src/main/java/com/introproventures/graphql/jpa/query/schema/impl/JpaPredicateBuilder.java

```diff
@@ -262,6 +265,19 @@ protected Predicate getDatePredicate(Path<? extends Date> root, PredicateFilter
         return null;
     }
 
+    private Predicate getUuidPredicate(Path<? extends UUID> field, PredicateFilter filter) {
```

Comment:
What about this implementation?
```
private Predicate getUUIDPredicate(final Path<UUID> root, final PredicateFilter filter) {

        final Predicate arrayValuePredicate = mayBeArrayValuePredicate(root, filter);

        if(arrayValuePredicate == null) {
            final UUID compareValue = (UUID) filter.getValue();

            if (filter.getCriterias().contains(EQ)) {
                return cb.equal(root, filter.getValue());
            }
            if (filter.getCriterias().contains(IN)) {
                final In<Object> in = cb.in(root);
                return in.value(compareValue);
            }
            if (filter.getCriterias().contains(NE)) {
                return cb.notEqual(root, compareValue);
            }
        }

        return arrayValuePredicate;
    }
```

Reply:
Or just add `In` criteria to the yours implementation.

Reply:
I'll gladly add the `In` Criteria.